diagnostic.md

```diff
@@ -36,13 +41,18 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/boston_cream
 ```
```

Comment:
this would be boston_cream's id not it's name. Unless you've set up your api really weird. But let's assume you didn't .